source/documentation/inference/__index.md

```diff
@@ -1054,8 +1054,8 @@ configuration still leaves something to be desired and will the subject of futur
 <p>We can create an instance of the OWL reasoner, specialized to the demo schema
   and then apply that to the demo data to obtain an inference model, as follows:</p>
 
-    Model schema = RDFDataMgr.loadModel("file:data/owlDemoSchema.owl");
-    Model data = RDFDataMgr.loadModel("file:data/owlDemoData.rdf");
+    Model schema = RDFDataMgr.loadModel("file:data/owlDemoSchema.xml");
+    Model data = RDFDataMgr.loadModel("file:data/owlDemoData.xml");
```

Comment:
Might be better to rename the example data files to have `.rdf` extensions and correct the links in the above paragraph instead?

Reply:
Sounds good. Please find it in the followup commit. I haven't generated the site to test it.

If you squash and rebase for PRs and I can push that instead. LMK?